src/main/java/org/commcare/formplayer/services/RestoreFactory.java

```diff
@@ -178,6 +183,9 @@ public void configure(AuthenticatedRequestBean authenticatedRequestBean, HqAuth
         log.info(String.format("configuring RestoreFactory from authed request with arguments " +
                         "username = %s, asUsername = %s, domain = %s",
                 username, asUsername, domain));
+        if (authenticatedRequestBean instanceof InstallRequestBean) {
+            storageFactory.configure(((InstallRequestBean)authenticatedRequestBean));
```

Comment:
this is already happening in the [AppInstallAspect](https://github.com/dimagi/formplayer/blob/dffa67cb818c8474eed05ec6983c8a0ad7483198/src/main/java/org/commcare/formplayer/aspects/AppInstallAspect.java#L43). I think the best approach for now is to add the `@AppInstall` annotation to whatever view it is missing from.